src/book/02-projects/03-hardware/01-overview.mdx

```diff
@@ -23,6 +23,19 @@ The arms and head of the NUgus robot use [Dynamixel MX64AR servos](http://emanua
 
 There are custom made mounting brackets milled from aluminium for the [flywheels](https://github.com/NUbots/NUware/blob/master/Igus/Current%20Design/stl_file/Flywheel_Bracket.stl), and servo-to-servo mounting brackets in the [neck](https://github.com/NUbots/NUware/blob/master/Igus/Current%20Design/stl_file/Head_Bracket.stl), hip, and ankle. There are also 3D printed cable clamps on the back of the [knees](https://github.com/NUbots/NUware/blob/master/Igus/Current%20Design/stl_file/Servo_cable_holder.stl), [ankle and hip blocks](https://github.com/NUbots/NUware/blob/master/Igus/Current%20Design/stl_file/Cable_Holder.stl).
 
+The servos have been modified to include a [1N6385 TranZorb](http://www.vishay.com/docs/88356/icte.pdf) connected in parallel to the servo motor.
+This is done to prevent induced EMF from overloading the electronics in the servo.
+EMF can be induced when the servos are manually moved without power being applied (i.e. grabbing the robot's leg and forcing the knee to bend).
+
+<Grid columns='1fr 2fr'>
+
+![Top view of the mounted TranZorb](./images/servo-tranzorb-1.png 'Top view of the mounted TranZorb. The two arms of the TranZorb are soldered to the motor terminals. Hot glue is added to stabilise the TranZorb against shocks and vibrations.')
+![Side view of the mounted TranZorb](./images/servo-tranzorb-2.png 'Side view of the mounted TranZorb. TranZorb arm is soldered on to the motor terminal along with the already existing motor connection (black cable)')
```

Comment:
This was more for the alt text, not the caption. Accessibility is important! (this is changed by changing the stuff inside the square brackets, ie [Top view of the mounted TranZorb].

Reply:
It's also quite useful in the caption.